src/entities/Updates/utils.ts

```diff
@@ -67,3 +69,11 @@ export const isBetweenLateThresholdDate = (dueDate?: Date) => {
 
   return Time().isBefore(newDueDate)
 }
+
+export function updateUrl(updateId: string, proposalId: string) {
```

Comment:
I prefer using verbs to name functions (getUpdateUrl). And I think we are doing it this way on the code, except some old functions.